collector/consumer/exporter/otelexporter/self_telemetry.go

```diff
@@ -6,20 +6,20 @@ import (
 	"sync"
 )
 
-var otelexporterGaugegroupsReceivedTotal = "kindling_telemetry_otelexporter_gaugegroups_received_total"
```

Comment:
Here I changed the self-telemetry metric, will this influence other components?

Reply:
No.